authority/migrations/0001_initial.py

```diff
@@ -24,10 +24,10 @@ class Migration(migrations.Migration):
                 ('approved', models.BooleanField(default=False, help_text='Designates whether the permission has been approved and treated as active. Unselect this instead of deleting permissions.', verbose_name='approved')),
                 ('date_requested', models.DateTimeField(default=datetime.datetime.now, verbose_name='date requested')),
                 ('date_approved', models.DateTimeField(null=True, verbose_name='date approved', blank=True)),
-                ('content_type', models.ForeignKey(related_name='row_permissions', to='contenttypes.ContentType')),
-                ('creator', models.ForeignKey(related_name='created_permissions', blank=True, to=settings.AUTH_USER_MODEL, null=True)),
-                ('group', models.ForeignKey(blank=True, to='auth.Group', null=True)),
-                ('user', models.ForeignKey(related_name='granted_permissions', blank=True, to=settings.AUTH_USER_MODEL, null=True)),
+                ('content_type', models.ForeignKey(related_name='row_permissions', to='contenttypes.ContentType', on_delete=models.CASCADE)),
```

Comment:
Migrations needed to be updated for Django 2.2 (they needed on_delete for all FKs)